.github/workflows/ci_build.yml

```diff
@@ -0,0 +1,18 @@
+name: CI Compilation testing
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { OS: alpine }
+    steps:
+      - uses: actions/checkout@v2
+      - name: Integration testing
+        run: |
+          docker build -f docker/Dockerfile.${{matrix.OS}} -t clightning-${{matrix.OS}} .
+          docker run clightning-${{matrix.OS}}
```

Comment:
With the entry point below, and missing the `bitcoin-cli` this'll just report an error about not being able to contact bitcoin and exit.

Reply:
We should be able to compile without `bitcoin-cli` or better, we must be able to compile without bitcoin-cli because we don't directly depend on bitcoin core.

In addition, I would like to avoid installing a bitcoin core in the image, because if we add a tor dependence, we could ship a docker image and the user can use it with docker-compose.